test/CMakeLists.txt

```diff
@@ -54,6 +56,17 @@ if(BUILD_TESTING)
             PASS_REGULAR_EXPRESSION "dgemm_\\|[ \t]+100000\\|"
     )
 
+    # Add test of config_file
+    add_test(NAME test_config_file COMMAND ${PROJECT_BINARY_DIR}/test/dgemm/test_dgemm)
```

Comment:
The test_dgemm only contains dgemm calls, but your `blas_symbols.conf` file does not have that symbol in it. Is this test really doing anything?